deploy/index.md

```diff
@@ -45,9 +45,9 @@ A CDN deployment of Protomaps has three parts:
 * [Deploy PMTiles on AWS](/deploy/aws)
 * [Deploy PMTiles with the Caddy web server](/deploy/server)
 
-## Deployement Comparison Chart
+## Deployment Comparison Chart
 
-_Cloudflare is recommended for budget minded beginners. AWS is recommended for advanced developers who are monetizing their map and can offset increased marginal costs or who require a faster map._
+_Cloudflare is recommended for budget minded beginners. AWS is recommended for advanced developers who require a faster map and/or can offset increased marginal cost with monetization._
```

Comment:
Maybe just "for commercial companies"?

Reply:
Fixed via https://github.com/protomaps/docs/pull/25/commits/f99baeb4f176a6e5aa6e1e1fe92d82fc9b32ade4